word_guess.rb

```diff
@@ -0,0 +1,198 @@
+require 'colorize'
+
+class WordGuess
+	attr_accessor :guesses_wrong, :win
+
+	def initialize
+    @correct_word = %w(H A N G M A N)
+    @guessing_array = %w(_ _ _ _ _ _ _)
+		@guesses_wrong = 0
+    @wrong_letters = []
+    @counter = "poop"
```

Comment:
I'm curious what this variable is being used for. It seems like it is being checked against "on" and "off" later in your class.